storm-client/src/jvm/org/apache/storm/utils/NimbusClient.java

```diff
@@ -96,6 +96,10 @@ public static NimbusClient getConfiguredClientAs(Map<String, Object> conf, Strin
         if (override != null) {
             return new NimbusClient(override);
         }
+        Map<String, Object> fullConf = Utils.readStormConfig();
```

Comment:
This is something I ran into trying to get the ThroughputVsLatency topology to run, so I could test the metrics.  It simply means that if you pass in an empty config that it will fill in the rest from the command line, storm.yaml, and defaults.yaml, just like when submitting a topology.